florist/app/jobs/hooks.tsx

```diff
@@ -47,3 +47,10 @@ export const usePost = () => {
 
     return { post, response, isLoading, error };
 };
+
+export function refreshJobsByJobStatus() {
```

Comment:
You can receive an array of statuses as a param here to keep the status information out of this file and only a the `jobs/page.tsx` file.

Reply:
Good idea!